produce/lib/produce/itunes_connect.rb

```diff
@@ -18,7 +18,7 @@ def run
     def create_new_app
       application = fetch_application
       if application
-        UI.success("[iTC] App '#{Produce.config[:app_identifier]}' already exists (#{application.apple_id}), nothing to do on App Store Connect")
+        UI.success("[ASC] App '#{Produce.config[:app_identifier]}' already exists (#{application.apple_id}), nothing to do on App Store Connect")
```

Comment:
As `ASC` is not really a common acronym, I think this should just be removed instead of replaced.

Reply:
Maybe it will be soon. 🤷‍♂️

Reply:
I agree with @janpio here. I would probably prefer to move it since "asc" is more known for meaning "ascending" which is actually what I thought this PR was about based on the title 🙃